apps/minimal-app/configuration/nrf54l15dk_nrf54l15_cpuapp/images/mcuboot/prj.conf

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Blecon Ltd
```

Comment:
For similar reasons to the ones above, I wonder if the MCUBoot image config should be shared across all apps, so that you can build any app and end up with the same MCUBoot build